### DISCUSSION TITLE
Discuss Liberating Structures

### DISCUSSION THREAD

**Discussion by avidrucker** [knowledge-for-good/knowledgeforgood#6] [General]
See [link here](https://www.liberatingstructures.com/)

**Comment by seekanddefine** [comment]
[When Liberating
Structures are part
of everyday
interactions, it is
possible to: ](Principles.pdf)

**Reply by avidrucker** [reply]
This link appears broken to me. Does it work for you?

**Reply by avidrucker** [reply]
I was able to inspect this discussion history, and it says the link is https://static1.1.sqspcdn.com/static/f/1272828/26988242/1461347577013/Principles.pdf?token=XWroLv%2BVh0z5osF2Y8og9G5%2FIZA%3D - is this the right one? 🤓

**Reply by seekanddefine** [reply]
Thank you!

**Comment by seekanddefine** [comment]
When should we schedule the liberating structures workshop? This Sat/Sun or next? Suggested times?

**Reply by seekanddefine** [reply]
Workshop was scheduled and completed. Open to do another as people are interested.
